packages/fluent/scss/utils/index.scss

```diff
@@ -1,23 +1,480 @@
-// Module meta
-$_kendo-module-meta: (
-    name: "utils",
-    dependencies: ()
+@use "../core" as *;
+
+@use "@progress/kendo-theme-utils/scss/index.import.scss" as * with (
+    $kendo-spacing: $kendo-spacing,
+    $kendo-font-sizes: $kendo-font-sizes,
+    $kendo-border-radii: $kendo-border-radii,
+    $kendo-elevation: $kendo-elevation,
+    $kendo-colors: $kendo-colors,
+    $kendo-enable-color-system: $kendo-enable-color-system
 );
 
+// Accessibility
+@mixin utils--accessibility--screen-readers() {
+    @include kendo-utils--accessibility--screen-readers();
+}
```

Comment:
why do we need to wrap each of these, instead of using them directly e.g `@include kendo-utils.mixin-name()` ? In general, this is error prone, as we need to maintain those mixins in all related packages.